requirements/static/ci/common.in

```diff
@@ -8,14 +8,16 @@ boto3>=1.17.67; python_version >= '3.6'
 boto>=2.46.0
 cassandra-driver>=2.0
 certifi>=2022.12.07
-cffi>=1.12.2
+cffi>=1.12.2, <= 1.14.6; python_version < '3.7'
+cffi>=1.15.1; python_version > '3.7'
 cherrypy>=17.4.1
 clustershell
 croniter>=0.3.0,!=0.3.22"; sys_platform != 'win32'
 dnspython
 docker
 etcd3-py==0.1.6 ; python_version >= '3.6'
 gitpython>=3.1.30; python_version >= '3.7'
+immutables>=0.19
```

Comment:
This is already 0.19 in the 3006.x branch, @MKLeb added it to the generated files instead of common.in, see 
```
david@david-XPS-15-9570:~/devcode/dgm_salt_master/salt/requirements$ grep -R immutables==0.19
static/ci/py3.10/pkgtests-windows.txt:immutables==0.19
static/ci/py3.10/lint.txt:immutables==0.19
static/ci/py3.10/pkgtests.txt:immutables==0.19
static/ci/py3.10/cloud.txt:immutables==0.19
static/ci/py3.7/lint.txt:immutables==0.19
static/ci/py3.7/cloud.txt:immutables==0.19
static/ci/py3.8/lint.txt:immutables==0.19
static/ci/py3.8/cloud.txt:immutables==0.19
static/ci/py3.9/lint.txt:immutables==0.19
static/ci/py3.9/cloud.txt:immutables==0.19
```

Reply:
Vaguely remember the lint issue, hence the change in common.in, since next change could overwrite the outputs for various python versions.

Reply:
Done